### PR TITLE
open-cluster-management/backlog#12616 mov upd lbl

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
#12616

File: console/cluster_label.adoc [hunk 1]
```diff
@@ -7,20 +7,18 @@ See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[La
...
 . Find your cluster in the _Clusters_ table.
-. Click the *Actions* icon to see more.
-. Click *Edit labels*.
-. From the _Edit labels_ dialog box, enter a label. Your entry might resemble the following content: `Key=Value`. 
-. Hit enter or space to add additional labels.
-. Click *Add*. Labels are saved when the *Add* button is clicked. 
-. Click *Done* to close the Edit labels dialog, when you are finished.
+. Click *Options* to see the options in the menu.
+. Select *Edit labels*.
+. In the "Edit labels" dialog box, enter a label. Your entry might resemble the following content: `Key=Value`.
```
Comment (swopebe) [inline console/cluster_label.adoc]: In the _Edit labels_ box...

As dialogue, this has always needed italicized.

File: console/cluster_label.adoc [hunk 2]
```diff
@@ -7,20 +7,18 @@ See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[La
...
-. From the _Edit labels_ dialog box, enter a label. Your entry might resemble the following content: `Key=Value`. 
-. Hit enter or space to add additional labels.
-. Click *Add*. Labels are saved when the *Add* button is clicked. 
-. Click *Done* to close the Edit labels dialog, when you are finished.
+. Click *Options* to see the options in the menu.
+. Select *Edit labels*.
+. In the "Edit labels" dialog box, enter a label. Your entry might resemble the following content: `Key=Value`. 
+. Press enter, add a comma, or add a space to add additional labels.
+. Click *Save* when you enter your last one. Labels are only saved after you click *Save*. 
+. Close the Edit labels pane, when you are finished.
```
Comment (swopebe) [inline console/cluster_label.adoc:17]: Here, as well.

File: console/cluster_label.adoc [hunk 3]
```diff
@@ -7,20 +7,18 @@ See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[La
...
-* If you want to update an existing label, you can reassign its key to a new value by adding a new label using the same key with a different value.
+* If you want to remove an existing label, click the *Remove* icon for the label that you want to remove in the list.
+* If you want to update an existing label, you can reassign its key to a new value by adding a new label using the same key with a different value. For example, you can change `Key=Value` by entering `Key=NewValue` to update the value of `Key`.
 
-*Note*: You can also edit a cluster label from a cluster details page. From the navigation menu, click *Automate infrastructure* > *Clusters*. 
-From the *Clusters* view, access a cluster's details page by clicking the cluster's name, then *Actions* > *Edit labels*.
+*Tip*: You can also edit a cluster label from the cluster details page. In the navigation menu, click *Infrastructure* > *Clusters*. 
+From the Clusters page, access a cluster's details page by clicking the name of the cluster. Select the *Edit* icon in the *Labels* section. The "Edit labels" dialog box is displayed.
```
Comment (swopebe) [inline console/cluster_label.adoc]: Please remove possessive

Comment (swopebe) [reply console/cluster_label.adoc]: Please remove quotations here and go back to original italicizing, as dialogue boxes are per style manual.

File: console/cluster_label.adoc (hunk 1 above)
Comment (cadawson) [reply console/cluster_label.adoc]: I thought so, but that is surprisingly not consistent with the style guide. I prefer italics, anyway.

File: console/cluster_label.adoc (hunk 3 above)
Comment (cadawson) [reply console/cluster_label.adoc]: I removed the double quotes. I was surprised to see that the style manual says use double quotes (see page 167): 
Titles of windows and
similar elements:
Dialog boxes
Notebooks
Notebook pages
Panes
Panels
Portals
Portlets
Windows
Wizards
Elements with headline-style capitalization: No
highlighting
Elements with sentence-style capitalization:
Double quotation marks
Open the Preferences
notebook.
In the User panel, type the
name of the user.
When you click the icon, the
"Open the remote database"
window opens.

**Review by cadawson — COMMENTED** [review]
Thank you for the review. Updates made.

**Comment by swopebe** [comment]
@cadawson I will look into the PDF, but I remember this from IBM with even the cloud-based manual and I memorized it. You may remember it Dita that we used the `wintitle` tag: **https://www.oxygenxml.com/dita/1.3/specs/langRef/technicalContent/wintitle.html

Without that tag to set this off nicely, we still followed the guidelines; never have used quotation marks in doc ever, but can you tell me your thoughts about changing it or why you did? Just want to see where the mix up is since this is how our doc has been and never have we changed it to "".

**Comment by swopebe** [comment]
Without the latest and easy-to-use manual that we had at IBM, we can also look here: https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

Here it actually says bold (no quotations) but we moved with all the content formatted for IBM Style. If you use "" in this topic and the rest of the doc is not using these, that presents an issue for the user. I would like to keep the doc the same across pillars, which was and is built with style manuals in mind.

**Comment by cadawson** [comment]
I am ok with that. It is strange that that supplementary style guide is different from the IBM guide. It should be set now. Thanks.

**Review by swopebe — APPROVED** [review]
thanks for pulling me in

File: console/cluster_label.adoc (hunk 2 above)
Comment (dockerymick) [reply console/cluster_label.adoc:17]: I agree add italics to so that it reads: _Edit labels_ pane

File: console/cluster_label.adoc [hunk 4]
```diff
@@ -7,20 +7,18 @@ See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[La
...
-* If you want to update an existing label, you can reassign its key to a new value by adding a new label using the same key with a different value.
+* If you want to remove an existing label, click the *Remove* icon for the label that you want to remove in the list.
+* If you want to update an existing label, you can reassign its key to a new value by adding a new label using the same key with a different value. For example, you can change `Key=Value` by entering `Key=NewValue` to update the value of `Key`.
 
-*Note*: You can also edit a cluster label from a cluster details page. From the navigation menu, click *Automate infrastructure* > *Clusters*. 
-From the *Clusters* view, access a cluster's details page by clicking the cluster's name, then *Actions* > *Edit labels*.
+*Tip*: You can also edit a cluster label from the cluster details page. In the navigation menu, click *Infrastructure* > *Clusters*. 
+From the Clusters page, access the details page for the cluster by clicking the name of the cluster. Select the *Edit* icon in the *Labels* section. The _Edit labels_ dialog box is displayed.
```
Comment (dockerymick) [inline console/cluster_label.adoc]: Is the edit icon a pencil or something? If no, I would suggest that it reads: Select **Edit** in the _Labels_ section. 

be sure to add italics around "Labels"

**Review by dockerymick — COMMENTED** [review]
Made a few comments, thanks for pulling me in Chris !

**Pull request merged**